plane/source/docs/quadplane-flying.rst

```diff
@@ -22,18 +22,27 @@ ground station to command a mode change.
 
 -  If you transition to :ref:`MANUAL <manual-mode>` then the VTOL motors
    will immediately stop. In the case of a tilt-rotor, the motors will immediately rotate to foward flight orientation.
+
+.. warning:: If you do not have sufficient airspeed, an immediate stall will occur! Since MANUAL mode is often setup as a reflex driven "bail-out", some users move, or remove this mode, and substitute QSTABLIZE or QLOITER as an alternative "bail-out" for a QuadPlane
+ 
 -  If you transition to any other fixed wing mode then the VTOL motors will
    continue to supply lift and stability until you have reached the
-   :ref:`ARSPD_FBW_MIN <ARSPD_FBW_MIN>` airspeed (or airspeed estimate if no airspeed sensor). In the case of tilt-rotors, the motors will tilt to :ref:`Q_TILT_MAX<Q_TILT_MAX>` to begin building forward airspeed for the transistion.
+   :ref:`ARSPD_FBW_MIN <ARSPD_FBW_MIN>` airspeed (or airspeed estimate if no airspeed sensor). In the case of tilt-rotors, the motors will tilt to :ref:`Q_TILT_MAX<Q_TILT_MAX>` to begin building forward airspeed for the transition. The VTOL motors will behave similar to that in QHOVER and will try to maintain present altitude through the transition. The forward thrust is controlled by the throttle stick in a manner similar to whatever fixed wing mode was entered. In FBWA, it is directly controlled, ie low stick is zero thrust and the QuadPlane will just hover. For tilt-rotors, stick positions below mid-stick will proportionately rotate VTOL motors back towards vertical, since that controls the forward thrust component. In FBWB/CRUISE, throttle stick controls forward thrust as in that mode, as a speed or throttle value, depending on whether or not an airspeed sensor is in use. 
```

Comment:
are there any caveats for operation without an airspeed sensor? 
e.g. QLOITER takeoff in strong headwind then switch to FBWA without flying around first; will the airspeed estimate be too low?

Reply:
biggest issue I have had is when transition AS is reached....ie a loooong time if flying into a headwind...as soon as you turn crosswind it will attain the aspeed...or if you waggle back and forth to allow the turn math to calculate a better synth airsped..

assist is a bitch without an arsd sensor...because synth as can occasionally radically miscalculate and throw you inot assist with an accompanying transition...I intend to add a note about both issues

Reply:
Thanks, that makes sense.